lib/_emerge/unmerge.py

```diff
@@ -556,21 +525,18 @@ def _pkg(cpv):
             writemsg_level("\n", noiselevel=-1)
 
     writemsg_level(
-        "\nAll selected packages: %s\n" % " ".join("=%s" % x for x in all_selected),
+        f"\nAll selected packages: {' '.join(f'={x}' for x in all_selected)}\n",
         noiselevel=-1,
     )
-
     writemsg_level(
-        "\n>>> "
-        + colorize("UNMERGE_WARN", "'Selected'")
-        + " packages are slated for removal.\n"
+        "\n>>> {} packages are slated for removal.\n".format(
+            colorize("UNMERGE_WARN", "'Selected'")
```

Comment:
Ah, I recall why I chose to do that. It was because of the `"'Selected'"` string having both double and single quote characters and wanted to avoid unnecessary escaping and edge cases. Thinking about about it, I can use the triple quote `"""` pattern to get around this and use it as an f-string.